src/ch/imedias/rsccfx/model/xml/SupporterHelper.java

```diff
@@ -65,28 +68,26 @@ public void saveSupporters(List<Supporter> supporters) {
    */
   public List<Supporter> getDefaultSupporters() {
     LOGGER.info("Loading default supporter list");
-    File supportersXmlFile =
-        new File(model.getPathToDefaultSupporters());
+    File supportersXmlFile;
+    try {
+      supportersXmlFile = new File(model.getPathToDefaultSupporters());
+    } catch (NullPointerException e) {
+      LOGGER.warning("Path to default supporters file is null! Returning null.");
+      return null;
+    }
     return getSupportersFromXml(supportersXmlFile);
   }
 
   private List<Supporter> getSupportersFromXml(File file) {
-    List<Supporter> supportersList = null;
-    try {
-      JAXBContext jaxbContext = JAXBContext.newInstance(Supporters.class);
-
-      Unmarshaller jaxbUnmarshaller = jaxbContext.createUnmarshaller();
-      Supporters supporters = (Supporters) jaxbUnmarshaller.unmarshal(file);
-
-      supportersList = supporters.getSupporters();
-    } catch (JAXBException e) {
-      e.printStackTrace();
-    }
-    return supportersList;
+    return getSupportersFromXml(fileToString(file));
```

Comment:
Since the code was essentially duplicated before with only minor changes, I decided to extract the parts that were different to "fileToString()" and use the similar method to avoid duplication